doc/ref_impl/cntt-ri2/chapters/chapter04.rst

```diff
@@ -0,0 +1,157 @@
+Operational Runbook
+===================
+
+Introduction
+------------
+
+This chapter documents the steps to deploy Kubernetes based Reference Implementation (RI-2) according to RA-2. The entire deployment has been tested in Anuket Labs as a part of the `Anuket Kuberef project <https://wiki.anuket.io/display/HOME/Kuberef>`__, that aims to deliver RI-2 based on RA-2 specifications. The Kuberef project stores all the code needed to deploy RI-2 and hence serves as a reference platform for CNF vendors to develop and test against. Currently, Kuberef supports deployments on both baremetal, as well as pre-provisioned infrastructure (for e.g. offered by baremetal providers like Equinix Metal, etc.).
+
+The entire installation is divided into two stages - Host provisioning and Kubernetes provisioning. Host provisioning is the operation of preparing a host before the software stack can be installed on them. This includes (and not limited to) installing an operating system, configuring network so that the hosts are reachable via SSH, configuring storage, etc. This stage can be skipped when using preprovisioned hardware, infrastructure providers, etc. The Kubernetes provisioning stage is agnostic to the host provisioning stage, in that there is no dependency between the installer used for the Kubernetes provisioning stage and any tools used in the host provisioning stage.
+
+Prerequisites
+-------------
+
+You need one physical server acting as a jump server along with minimum of two additional servers on which RI-2 will be deployed. Please refer to `Chapter 3 <./chapter03.md>`__ for detailed information on the server and network specifications.
```

Comment:
this won't work anymore. You could simply run linkcheck to detect all of them.

Reply:
@collivier: fixed. the last commit switched to references where possible and passed linkcheck.